.github/workflows/nginx_test.yml

```diff
@@ -0,0 +1,20 @@
+name: Nginx test
+
+on: [push, pull_request]
+
+jobs:
+
+  test:
+    name: Run compose tests
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Run test
+      run: |
+        sudo apt update
+        sudo apt install -y bats
+        {
+          cd nginx
+          make test
+        }
```

Comment:
This is a mistake on my end, I meant to use `()` (subshell) instead of `{}` (group command)

It's to make relative navigation slightly easier to keep track of. Imagine we add tests for haproxy in the same file: instead of doing

```bash
cd nginx
# do tests...
cd ..

cd haproxy
# do tests...
cd ..
```
It's possible to
```bash
(
  cd nginx
  # do tests...
)
(
  cd haproxy
  # do tests...
)
```
This is a simple navigation so it's easy to follow either way, but I figure it's better to be proactive if the complexity creeps up

Reply:
works for me - thanks for the explanation !